easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-3.2-gimkl-2.11.5.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'EB_OpenFOAM'
+
+name = 'OpenFOAM-Extend'
+version = '3.2'
+
+homepage = 'http://www.extend-project.de/'
+description = """OpenFOAM is a free, open source CFD software package. 
+OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+involving chemical reactions, turbulence and heat transfer, 
+to solid dynamics and electromagnetics.""" 
+
+toolchain = {'name': 'gimkl', 'version': '2.11.5'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Unofficial-Extend-Project-Mirror/foam-extend-foam-extend-%(version)s/archive']
+sources = ['v%(version)s.tar.gz']
+
+patches = [
+    'OpenFOAM-Extend-%(version)s-ParMGridGen.patch',
+    'OpenFOAM-Extend-3.1_build-qa.patch',
+    'OpenFOAM-Extend-3.1_comp-mpi.patch',
+    'OpenFOAM-Extend-3.1_skip-ThirdParty-OpenMPI.patch',
+]
+
+dependencies = [
+#    ('hwloc', '1.11.0', '', ('GNU', '4.9.3-2.25')),
```

Comment:
OpenFOAM-Extend needs it. 

I don't want to add `hwloc` with the `gimkl` toolchain because it's pointless. The `GCC`/`GCCCore`/`GNU` toolchain suffices. No need to generate a bunch of redundant easyconfigs.